cache/cache_test.go

```diff
@@ -1,84 +1,55 @@
-package cache
+package cache_test
```

Comment:
same as https://github.com/newrelic/infra-integrations-sdk/pull/59#discussion_r181799985

Reply:
To be consistent with the previous work from the team, the reason is that this way you are "forced" to test only against public interfaces.